doc/Language/objects.pod6

```diff
@@ -296,6 +296,27 @@ Method names can be resolved at runtime with the C<.""> operator.
     A.new."$name"().say;
     # OUTPUT: «(Any)␤»
 
+The syntax used to update C<$.notes> changed in this section with respect
+to the previous L<#Attributes> section. Instead of:
+
+    $vacation.notes = 'Pack hiking gear and sunglasses!';
+
+we now:
+
+    $trip.notes("First steps");
+
+Overriding the default auto-generated accessor means it is no longer
+available to provide a mutable container on return, and the new update
+method is called to set the attribute rather than an assignment.
+Using a method call is the preferred approach to adding computation
+and logic to the update of an attribute.  Many modern languages can
+update an attribute by overloading assignment with a “setter” method.
+While Perl 6 can overload the assignment operator for this purpose
+with a
+L<C<Proxy>|https://github.com/perl6/roast/blob/master/S12-attributes/mutators.t>
+object, overloading assignment to set attributes with complex logic is
+currently discouraged as weaker OO design.
```

Comment:
Expand to "object oriented". Maybe add a reference. Where is it discouraged?

Reply:
Hopefully addressed in latest commit.